pallets/parachain-staking/src/inflation.rs

```diff
@@ -26,12 +26,12 @@ use substrate_fixed::transcendental::pow as floatpow;
 use substrate_fixed::types::I64F64;
 
 const SECONDS_PER_YEAR: u32 = 31557600;
-const SECONDS_PER_BLOCK: u32 = 12;
+const SECONDS_PER_BLOCK: u32 = 6;
```

Comment:
Is it fine to hardcode this value in  a pallet (vs for example, for us, passing the `MILLISECONDS_PER_BLOCK` defined in the runtime as a config value)? Are all substrate chains using this pallet expected to have a 6 second block? By the way, this value is used to compute `BLOCKS_PER_YEAR` which seems to only be used on the genesis config to  compute some inflation related values. Should we still use `BLOCKS_PER_YEAR` there (vs slots per year)?

Reply:
No it's technical debt, I actually refactored the code so that these constants are no longer used but I forgot to delete them